app/controllers/candidate_interface/after_sign_in_controller.rb

```diff
@@ -22,6 +23,12 @@ def interstitial
 
   private
 
+    def redirect_to_path_if_path_params_are_present_and_valid
+      return false if params[:path].blank? || valid_paths.exclude?(params[:path])
+
+      redirect_to path_params
```

Comment:
Brakeman is complaining about this method, but it is protected via the valid_paths logic above.

Reply:
I wonder in that case if this logic needs to be reversed so that it whitelists valid paths?

Reply:
Yup good point, must have been on autopilot 🤦